src/grimp/application/usecases.py

```diff
@@ -14,6 +15,8 @@
 from ..domain.valueobjects import DirectImport, Module
 from .config import settings
 
+N_CPUS = multiprocessing.cpu_count()
```

Comment:
🤔 I wonder what the overhead of this is, probably tiny but it might make sense to calculate it at runtime, only if we need it.

Maybe worth tweaking next time we're in this file.